Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/domain/TaskScheduler.kt

```diff
@@ -0,0 +1,26 @@
+package com.mentalmachines.droidcon_boston.domain
+
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+
+/**
+ * Didn't wanted to add dependency for thread management like Rx or coroutine only for twitter
```

Comment:
Helpful comment, thanks! :)

Reply:
adding either if it would have made your life easier also would have been fine...